frontend/src/pages/instructor/test-components.tsx

```diff
@@ -107,44 +58,88 @@ export default function InstructorHome({
         />
       </Head>
 
-      <div>
-        <PopUpProvider>
-          <div className="bg-surface_bg_primary">
-            <MobileDetailView
-              backgroundColor="bg-surface_bg_highlight"
-              className="px-[16px] pt-[24px]"
-              headerContent={
-                <div className="flex items-center gap-[12px]">
-                  <h3 className="text-typeface_primary text-h3">
-                    Class Schedule
-                  </h3>
-                </div>
-              }
-            >
-              <div className="pb-[81px]">
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus
-                pellentesque auctor scelerisque. Quisque dictum nunc ut vehicula
-                vehicula. Ut at mi eu nisi tempor vehicula non at lacus. Sed
-                nunc nibh, finibus quis maximus ac, euismod a risus. Duis cursus
-                id urna at gravida.erra massa vel tempor tempus. Nunc gravida
-                cursus venenatis. Integer placerat eleifend nisi a commodo.
-                Phasellus non fermentum leo, at posuere purus. Maecenas rhoncus
-                pulvinar neque, et condimentum odio laoreet vitae. Nulla
-                molestie dui vitae faucibus dapibus.
-              </div>
-              <AudioPlayer
-                audioSrc="https://actions.google.com/sounds/v1/ambiences/coffee_shop.ogg"
-                title="Coffee Shop"
-              ></AudioPlayer>
-              <AudioPlayerMobile
-                audioSrc="https://actions.google.com/sounds/v1/ambiences/coffee_shop.ogg"
-                title="Coffee Shop"
-              ></AudioPlayerMobile>
-            </MobileDetailView>
+      <PopUpProvider>
```

Comment:
what is this ?
if it's testing classmodecontent you could've also done it by changing classmodecontainer (that's what that's for); you can access that by setting a session to your current time and entering the class